tests/test_tradehub_authenticated_client.py

```diff
@@ -93,9 +93,14 @@ def test_unbond_tokens(self):
         txn_message: dict = types.BeginUnbondingTokensMessage(delegator_address=self._wallet.address,
                                                               validator_address=self.validator_address,
                                                               amount=types.AmountMessage(amount='0.1', denom='swth'))
+        time.sleep(2)
         result: dict = self.authenticated_client.unbond_tokens(message=txn_message)
 
-        self.assertDictStructure(expect=expect, actual=result)
+        if 'code' in result:
+            self.assertEqual(first='too many unbonding delegation entries for (delegator, validator) tuple: failed to execute message; message index: 0',
```

Comment:
Do you know how many unbonding delegations are allowed?

Reply:
Just tried this on a fresh account, it looks like it is 7 times.

https://switcheo.org/account/tswth1rjr3nm2xcyr6psseksk8vhpc4n33htntuhmdfm?net=test